CONTRIBUTING.md

```diff
@@ -107,21 +144,66 @@ Tests can be run as follows:
 poetry run bash scripts/test-coverage-xml.sh
 ```
 
-Please note that it is good practice to run the above commands before submitting any Pull Request: The CI GitHub Action
+Please note that it is good practice to run the above commands before submitting
+any Pull Request: The CI GitHub Action
 will run it anyway, so might as well catch the errors locally.
 
 ### 🚨 Reporting a Vulnerability
 
-If you think you found a vulnerability, and even if you are not sure about it, please report it right away by sending an
-email to: support@zenml.com. Please try to be as explicit as possible, describing all the steps and example code to
+If you think you found a vulnerability, and even if you are not sure about it,
+please report it right away by sending an
+email to: support@zenml.com. Please try to be as explicit as possible,
+describing all the steps and example code to
 reproduce the security issue.
 
 We will review it thoroughly and get back to you.
 
-Please refrain from publicly discussing a potential security vulnerability as this could potentially put our users at
-risk! It's better to discuss privately and give us a chance to find a solution first, to limit the potential impact
+Please refrain from publicly discussing a potential security vulnerability as
+this could potentially put our users at
+risk! It's better to discuss privately and give us a chance to find a solution
+first, to limit the potential impact
 as much as possible.
 
+h2: ## Coding Conventions
+
+## Coding Conventions
+
+The code within the repository is structured in the following way - 
+the most relevant places for contributors are highlighted with a `<-` arrow:
+
+```
+├── .github           -- Definition of the GH action definitions
```

Comment:
```suggestion
├── .github           -- Definition of the Github Action definitions
```

A bit circular. Maybe 'def of the gh action code' or workflows or something like that?